Documentation of `componentOutputs` looks not enough.

## Summary

When `render(ComponentType)` includes `componentOutputs` in its option, TypeScript inference seems broken.

## Repro

BTW, I'm preparing a bugfix for this. This is a spec to reproduce the bug.

```ts
describe('componentOutputs', () => {
  it('invokes given callback when an output is emitted', async () => {
    @Component({ template: `` })
    class TestFixtureComponent {
      @Output() event = new EventEmitter<void>();
      emitEvent() {
        this.event.emit();
      }
    }

    const spy = jest.fn();
    const { fixture } = await render(TestFixtureComponent, {
      componentOutputs: { event: spy },
    });

    fixture.componentInstance.emitEvent();

    expect(spy).toHaveBeenCalled();
  });
});
```

```
           projects/testing-library/tests/render.spec.ts:170:31 - error TS2769: No overload matches this call.
             Overload 1 of 2, '(component: Type<TestFixtureComponent>, renderOptions?: RenderComponentOptions<TestFixtureComponent, typeof import("/Users/lacolaco/works/angular-testing-library/node_modules/@testing-library/dom/types/queries")> | undefined): Promise<...>', gave the following error.
               Type 'Mock<any, any>' is missing the following properties from type 'EventEmitter<void>': emit, subscribe, closed, currentObservers, and 16 more.
             Overload 2 of 2, '(template: string, renderOptions?: RenderTemplateOptions<WrapperComponent, {}, typeof import("/Users/lacolaco/works/angular-testing-library/node_modules/@testing-library/dom/types/queries")> | undefined): Promise<...>', gave the following error.
               Argument of type 'typeof TestFixtureComponent' is not assignable to parameter of type 'string'.
       
           170     const { fixture } = await render(TestFixtureComponent, {
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           171       componentOutputs: { event: spy },
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           172     });
               ~~~~~~
```

<img width="1289" alt="image" src="https://user-images.githubusercontent.com/1529180/218715761-d803f0b6-2acc-42a3-a24f-5f4c75740c0d.png">

--- a/apps/example-app/src/app/examples/02-input-output.spec.ts
+++ b/apps/example-app/src/app/examples/02-input-output.spec.ts
@@ -6,8 +6,10 @@ test('is possible to set input and listen for output', async () => {
   const sendValue = jest.fn();
 
   await render(InputOutputComponent, {
-    componentProperties: {
+    componentInputs: {
       value: 47,
+    },
+    componentOutputs: {
       sendValue: {
         emit: sendValue,
       } as any,

--- a/projects/testing-library/src/lib/models.ts
+++ b/projects/testing-library/src/lib/models.ts
@@ -221,9 +221,12 @@ export interface RenderComponentOptions<ComponentType, Q extends Queries = typeo
    * {}
    *
    * @example
+   * const sendValue = (value) => { ... }
    * const component = await render(AppComponent, {
    *  componentOutputs: {
-   *    send: (value) => { ... }
+   *    send: {
+   *      emit: sendValue
+   *    }
    *  }
    * })
    */

--- a/projects/testing-library/tests/render.spec.ts
+++ b/projects/testing-library/tests/render.spec.ts
@@ -8,6 +8,8 @@ import {
   APP_INITIALIZER,
   ApplicationInitStatus,
   Injectable,
+  EventEmitter,
+  Output,
 } from '@angular/core';
 import { NoopAnimationsModule, BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { TestBed } from '@angular/core/testing';
@@ -152,6 +154,28 @@ describe('removeAngularAttributes', () => {
 
     expect(document.querySelector('[ng-version]')).not.toBeNull();
     expect(document.querySelector('[id]')).not.toBeNull();
+  });
+});
+
+describe('componentOutputs', () => {
+  it('should set passed event emitter to the component', async () => {
+    @Component({ template: `` })
+    class TestFixtureComponent {
+      @Output() event = new EventEmitter<void>();
+      emitEvent() {
+        this.event.emit();
+      }
+    }
+
+    const mockEmitter = new EventEmitter<void>();
+    const spy = jest.spyOn(mockEmitter, 'emit');
+    const { fixture } = await render(TestFixtureComponent, {
+      componentOutputs: { event: mockEmitter },
+    });
+
+    fixture.componentInstance.emitEvent();
+
+    expect(spy).toHaveBeenCalled();
   });
 });
 

